juniper_official/Solutions/Pfe-wedge-detection/eachip-pattern-set.rule

```diff
@@ -0,0 +1,489 @@
+                description "The \"ucode sb data error\" message is caused by a transient hardware issue, which is automatically repaired. This error is caused by a transient hardware defect. Local microcode sb(single-bit) or db(double-bit) memory errors are repaired from the shadow memory space.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "If few occurrences, no action.  If numerous occurrences, then contact JTAC.";
+                }
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT035 {
+                event-id EACHIP_EVENT035;
+                filter "fpc%{NUMBER:fpc_number}.*MQSS.*FO.*Parity error detected for output buffer control";
+                description "The \"Parity error detected for output buffer control\" message reports a transient hardware error in EACHIP Fabric output block. This is due to a transient hardware issue within the EACHIP Fabric output block.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "If only a few errors occur, then clear the Alarm.  Else, contact JTAC.";
+                }
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT036 {
+                event-id EACHIP_EVENT036;
+                filter "fpc%{NUMBER:fpc_number}.*MQSS.*FI.*Parity error detected for bank.*of CP table";
+                description "The \"Parity error detected for bank .. of CP table\" message reports a hardware parity error detected in the EACHIP Fabric input block. This is due to a transient hardware SRAM error within EACHIP Fabric input block.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "If only a few errors occur, then clear the Alarm.  Else, contact JTAC.";
+                }
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT037 {   
+                event-id EACHIP_EVENT037;
+                filter "fpc%{NUMBER:fpc_number}.*HMM.*Lane.*TX RLL event ";
+                description "The \"TX RLL event\" message reports Run Length Limit (RLL) event on a Hybrid Memory Link. This indicates a Run Length Limit event on scrambled links and Junos OS will automatically retry to recover. Successful recovery will be confirmed with the second debug message with State: values between 0x20 - 0x23.​ A response packet with a benign WARNING state is received from HMC - State: 0x20";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "No action if only a few occurrences.  If seen repeatedly, contact JTAC.";
```

Comment:
And here..

Reply:
@ConstrainedRandomVar , added trigger based on count